casper/src/test/scala/coop/rchain/casper/MergeabilityRules.scala

```diff
@@ -263,7 +263,9 @@ trait MergeabilityRules {
         expectOne(right) { right =>
           val (a, b) = if (left.maybeCardinality == Some(Linear)) (left, right) else (right, left)
           assert(
-            a.maybeCardinality == Some(Linear) && (b.maybeCardinality == Some(Linear) || hasMatch(
+            a.maybeCardinality == Some(Linear) && (b.maybeCardinality == Some(Linear) || b.maybeCardinality == Some(
+              Peek
+            ) || hasMatch(
```

Comment:
this reads `(a linear and b linear) or b peek`.
a doesn't matter it the or'ed case?